package.json

```diff
@@ -43,13 +43,19 @@
     "bates"
   ],
   "devDependencies": {
+    "@ava/typescript": "^1.1.1",
+    "@types/node": "^14.14.31",
+    "@types/seedrandom": "^2.4.28",
     "ava": "^3.5.0",
     "babel-cli": "^6.26.0",
     "babel-preset-env": "^1.7.0",
     "del-cli": "^1.1.0",
     "dev-cli": "^0.0.0",
     "esm": "^3.2.25",
     "standard": "^14.3.3",
+    "ts-node": "^9.1.1",
+    "tsc": "^1.20150623.0",
```

Comment:
shouldn't need to add `tsc` directly; it comes bundled with `typescript`.

Reply:
Noted